src/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -353,46 +353,58 @@ def get_beacon_config(
         "--metrics-allow-origin=*",
         "--metrics-port={0}".format(BEACON_METRICS_PORT_NUM),
         # ^^^^^^^^^^^^^^^^^^^ METRICS CONFIG ^^^^^^^^^^^^^^^^^^^^^
+        # "--allow-insecure-genesis-sync",
```

Comment:
can we remove the line if we are not using it?

Reply:
Ideally we want to enable both checkpoint sync and insecure genesis sync flag for lighthouse: https://github.com/sigp/lighthouse/issues/5054

I left it commented out while they discuss this issue.

Reply:
Oh, in my head adding it back is as easy and I don't like dead code lying around. Will leave this upto you.

Reply:
Its not dead code, is future code lol

Reply:
So I won't have to look it up what the flag is once this feature is enabled.

Reply:
Added an extra comment above this "dead" code. Otherwise I might forget about this.

Reply:
Alright, to me "dead code" is anything that doesn't get executed. As long as this gets cleaned up soon I can live with it!